index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/master.css">
+    <title>Apple-clone</title>
+  </head>
+
+  <body>
+    <img class="bg-img" src="images/your_verse_hero.jpg" alt="">
+
+    <div class="navbar row">
+      <button><img class="logo" src="images/logo.png" alt=""></button>
```

Comment:
Do you really need to use button tags within a nav? i think its better to use Anchor tag with list items.